### DISCUSSION TITLE
Version 1.1.2 release date?

### DISCUSSION THREAD

**Discussion by rafaelbenavent** [serverpod/serverpod#1439] [General]
I'm looking to use NOT IN and IN expressions for queries. I see it's in master branch. Does anyone know when it will be shipped?

**Comment by SandPod** [comment]
Hey! We are working hard on finishing the features that are included in our next release (1.2).

You can follow the progression on our project board here: https://github.com/orgs/serverpod/projects/4

**Reply by rafaelbenavent** [reply]
Thanks! v1.2 seems very ambitious! Have you considered releasing more frequently? I'm sure a lot of users could benefit from some of the already developed features. Could help increase interest in this project!

**Reply by SandPod** [reply]
Yes, I agree with you :) We are aiming to move more towards a continuous delivery model after the release of v1.2.
